src/theme/components/socialmedialinks.ts

```diff
@@ -0,0 +1,185 @@
+// maybe set the border to always be on, but invisible if prop is set to "none"?
```

Comment:
The default is "none" so the way you have it now seems right. Unless if there's something else that's bothering you about the implementation?

Reply:
This was written under the assumption that the border would add to the overall size of each link. If it does, and a dev wants to add borders under some circumstances but not others, then the sizing will change. This seems potentially problematic. Thus, if the border is always there, but invisible when set to "none," the sizing will stay consistent. 

But the question is moot if the border is meant to be _inside_ the div.